README.md

```diff
@@ -266,3 +268,30 @@ Please check out common DSC Community [contributing guidelines](https://dsccommu
 - [CMBoundaryGroups_Present](Source\Examples\Resources\CMBoundaryGroups\CMBoundaryGroups_Present.ps1)
 - [CMBoundaryGroups_Include](Source\Examples\Resources\CMBoundaryGroups\CMBoundaryGroups_Include.ps1)
 - [CMBoundaryGroups_Exclude](Source\Examples\Resources\CMBoundaryGroups\CMBoundaryGroups_Exclude.ps1)
+
+### CMAssetIntelligencePoint
+
+- **[String] SiteCode** _(Key)_: Specifies the Site Code for the Configuration
+  Manager site.
+- **[String] SiteServerName** _(Required)_: Specifies the Site Server to install
```

Comment:
remove SiteServerName as Required.
move SiteCode to Required
add IsSingleInstance as Key an example in this resource is in CMClientStatusSettings

Reply:
made required changes and pushed